src/main/java/seedu/address/model/CustomerManager.java

```diff
@@ -36,7 +36,7 @@ public Customer getCustomer(int customerId) {
      * @param customerId customer unique id.
      */
```

Comment:
Since you have extended EntityManager<T>. when u do getPersonList(), u should get a list of customer so may not need to typecast. can you check if typecasting  is still required?

Reply:
completed.